src/accelerate/tracking.py

```diff
@@ -180,11 +172,10 @@ class TensorBoardTracker(GeneralTracker):
 
     @on_main_process
     def __init__(self, run_name: str, logging_dir: Union[str, os.PathLike], **kwargs):
-        if is_tensorboard_available():
-            try:
-                from torch.utils import tensorboard
-            except ModuleNotFoundError:
-                import tensorboardX as tensorboard
```

Comment:
This will be called to assume `tensorboard` is available, and if it's not we should raise the import error in the case of a user is trying to pass their own `TensorBoardTracker` instance to the `Accelerator`